packages/tags/src/Tags.ts

```diff
@@ -104,9 +110,27 @@ export class Tags extends FocusVisiblePolyfillMixin(SpectrumElement) {
         this.rovingTabindexController.clearElementCache();
     }
 
+    private async changeDefaultFocus(event: Event): Promise<void> {
+        await 0;
+        if (event.defaultPrevented) {
+            // No deleting when user prevents it.
+            return;
+        }
+        const items = [...this.tags] as Tag[];
```

Comment:
This should already be an array. Is there a special benefit for copying it like this?